spec/cypress/integration/CipMentorMaterials.feature

```diff
@@ -19,7 +19,7 @@ Feature: Core Induction Programme mentor materials
 
   Scenario: Admins can edit mentor materials
     Given I am logged in as "admin"
-    Given mentor_material was created as "with_mentor_material_part" with id "a4dc302c-ab71-4d7b-a10a-3116a778e8d5"
+    And mentor_material was created as "with_mentor_material_part" with id "a4dc302c-ab71-4d7b-a10a-3116a778e8d5"
```

Comment:
Nice

Reply:
clearly the most important change in this PR